### PR TITLE
Introduce sanitizeHtml to sanitize markdown content

### PR DESCRIPTION
**Pull request by JanAckermann**
## Description
Bugfix: Prevent cross-site scripting attack while displaying space description

We've added a new package that strips out possible XSS attack code while displaying the space description

## Related Issue

- Fixes https://github.com/owncloud/web/issues/6526

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: packages/web-runtime/package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "fuse.js": "^6.5.3",
     "lodash-es": "^4.17.21",
     "luxon": "^2.3.0",
+    "marked": "^4.0.12",
```
Comment (pascalwengerter) [inline packages/web-runtime/package.json:19]: By adding this here, you should be able to remove the (now duplicate) dependency in the markdown extension itself:)

Comment (JanAckermann) [reply packages/web-runtime/package.json:19]: shouldn't extensions be independent ?

Comment (kulmann) [reply packages/web-runtime/package.json:19]: As long as we have the extensions in the monorepo I'm fine with maintaining dependencies centralized if they are used in multiple packages. That's what we're already doing, so I'm fine with what Pascal proposed. We'll have to have a close look at dependencies anyway when we split out extensions at some point.

**Review by pascalwengerter — APPROVED** [review]
LGTM 🧽

**Review by C0rby — APPROVED** [review]
Looks good and seems to be working. I couldn't set off any XSS payload I tried. :muscle:

**Review by kulmann — APPROVED** [review]

**Comment by ownclouders** [comment]
Results for <strong>oCISTrashbinUploadMoveJourney</strong> https://drone.owncloud.com/owncloud/web/23265/69/1
The following scenarios passed on retry:
- webUITrashbinRestore/trashbinRestore.feature:279
- webUITrashbinRestore/trashbinRestore.feature:46

**Pull request merged**